ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -284,7 +299,11 @@ export default defineComponent({
     }
 
     const localState = reactive({
+      ownersGroupId: showGroups.value
+        ? getGroupForOwner(props.editHomeOwner?.id)?.groupId
+        : null,
       owner: { ...defaultHomeOwner },
+      ownerGroupId: undefined,
```

Comment:
Why not `''` if it is of string type elsewhere?

Reply:
This is so we can use `OR ||` operator to set a default value easier.

Reply:
🤔 Where you used `OR` operator is was also defined as a string though, so i don't follow exactly.
`(localState.ownerGroupId as string) || '1'`

Reply:
Updated to remove type.